apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/converter/db/TxReceiptRowMapper.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.converter.db;
+
+import com.apollocurrency.aplwallet.api.v2.model.TxReceipt;
+import com.apollocurrency.aplwallet.apl.core.app.AplException;
+import com.apollocurrency.aplwallet.apl.core.signature.Signature;
+import com.apollocurrency.aplwallet.apl.core.signature.SignatureParser;
+import com.apollocurrency.aplwallet.apl.core.signature.SignatureToolFactory;
+import com.apollocurrency.aplwallet.apl.core.transaction.TransactionType;
+import com.apollocurrency.aplwallet.apl.core.transaction.TransactionTypeFactory;
+import com.apollocurrency.aplwallet.apl.core.transaction.UnsupportedTransactionVersion;
+import com.apollocurrency.aplwallet.apl.core.transaction.messages.MessageAppendix;
+import com.apollocurrency.aplwallet.apl.core.utils.Convert2;
+import com.apollocurrency.aplwallet.apl.crypto.Convert;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Singleton
+public class TxReceiptRowMapper implements RowMapper<TxReceipt> {
```

Comment:
What is a difference between plain Transaction, TransactionEntity and TxReceipt ?

Reply:
TxReceipt  - Transaction receipt. The lightweight and compact representation of the blockchain transaction. It doesn't support the transaction type conversion. The description added to apollo-api2-v2.yaml